### PR TITLE
Add message if no bonuses

### PR DESCRIPTION
**Pull request by Tamila-Bodnar**

### CODE REVIEW

File: src/app/core/services/toaster.service.ts [hunk 1]
```diff
@@ -19,9 +19,18 @@ export class ToasterService {
...
   }
 
-  public showShow(message: string, title: string): void {
+  public showShow(
+    message: string,
+    title: string,
+    toastrCustomClass?: string,
+    toastrCustomTitle?: string,
+    toasterCustomMessage?: string,
+  ): void {
```
Comment (rustamItCraft) [inline src/app/core/services/toaster.service.ts:28]: You had better use ...ar, instead of three optional parameters.

File: src/app/features/add-bonus/add-bonus-form/add-bonus-form.component.spec.ts [hunk 2]
```diff
@@ -13,7 +13,13 @@ describe('AddBonusFormComponent', () => {
   beforeEach(async () => {
     await TestBed.configureTestingModule({
       declarations: [AddBonusFormComponent],
-      imports: [RouterTestingModule, TranslateModule.forRoot(), MatAutocompleteModule, HttpClientModule, ToastrModule.forRoot(),],
+      imports: [
+        RouterTestingModule,
```
Comment (rustamItCraft) [inline src/app/features/add-bonus/add-bonus-form/add-bonus-form.component.spec.ts:17]: Why did you add so many spaces? remove them please

File: src/app/features/add-bonus/add-bonus-form/add-bonus-form.component.ts [hunk 3]
```diff
@@ -69,22 +69,21 @@ export class AddBonusFormComponent implements OnInit {
...
-        bonusDescription: bonus.description,
-        bonusTags: bonus.tags,
-        bonusTitle: bonus.title,
-        phone: bonus.phone,
-        start: bonus.dateStart,
-        end: bonus.dateEnd,
-      }
-    )
+  public fillPage(bonus: IBonus): void {
+    // TODO: fill all values
```
Comment (rustamItCraft) [inline src/app/features/add-bonus/add-bonus-form/add-bonus-form.component.ts:73]: Again useless space

File: src/app/shared/components/bonus-list-container/bonus-list-container.component.ts [hunk 4]
```diff
@@ -32,6 +32,17 @@ export class BonusListContainerComponent implements OnInit, OnDestroy {
       (data: IBonus[]) => {
         if (data) {
           this.bonuses = data;
+          if (this.bonuses.length === 0) {
+            setTimeout(() => {
```
Comment (rustamItCraft) [inline src/app/shared/components/bonus-list-container/bonus-list-container.component.ts:36]: Why did you use settimout? 
Also you didn't translate message in toaster

File: src/app/features/add-bonus/add-bonus-form/add-bonus-form.component.spec.ts (hunk 2 above)
Comment (Tamila-Bodnar) [reply src/app/features/add-bonus/add-bonus-form/add-bonus-form.component.spec.ts:17]: It is prettier

File: src/app/features/add-bonus/add-bonus-form/add-bonus-form.component.ts (hunk 3 above)
Comment (Tamila-Bodnar) [reply src/app/features/add-bonus/add-bonus-form/add-bonus-form.component.ts:73]: It's not me.

**Review by umilanovich — APPROVED** [review]
